views/_email-content.ejs

```diff
@@ -0,0 +1,70 @@
+<table border="0" cellpadding="0" cellspacing="0" width="122">
+    <tr>
+
+    <% for(var i=0; i < userData[participantIndex].users.length; i++) { %>
```

Comment:
Addressing the changes here https://github.com/drewww/unhangout/commit/0e2be9fd9d5a701c9c53ff853b1f82b285975688